proto/cosmos/accounts/defaults/base/v1/base.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package cosmos.accounts.defaults.base.v1;
+
+option go_package = "cosmossdk.io/x/accounts/defaults/base/v1";
+
+// MsgInit is used to initialize a base account.
+message MsgInit {
+  // pub_key defines the secp256k1 pubkey for the account.
+  bytes pub_key = 1;
+}
+
+// MsgInitResponse is the response returned after base account initialization.
+// This is empty.
+message MsgInitResponse {}
+
+// MsgSwapPubKey is used to change the pubkey for the account.
+message MsgSwapPubKey {
+  // new_pub_key defines the secp256k1 pubkey to swap the account to.
```

Comment:
Does this have to be secp256k1? How would an account use other curves?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/19395/files#diff-85ce529f991002c6567b96f005a1faaca17e41640da48585e9382e154cb3cfb5R77

^ currently you can implement whatever auth mechanism for your account, the developer is just required to make its own account implementation accept a MsgAuthenticate, then how authentication happens it's up to the dev, he can use curves or w/e. 

This implementation of baseaccount assumes the authentication method is always secp256k1, this could be abstracted to support other authentication methods, but I would handle this in a separate PR.